pkg/security/common/containerutils/utils.go

```diff
@@ -7,13 +7,14 @@
 package containerutils
 
 import (
-	"crypto/sha256"
-	"fmt"
 	"regexp"
 )
 
-// ContainerIDPatternStr is the pattern of a container ID
-var ContainerIDPatternStr = fmt.Sprintf(`([[:xdigit:]]{%v})`, sha256.Size*2)
+// ContainerIDPatternStr defines the regexp used to match container IDs
```

Comment:
Something I didn't get before sorry, we already have a `pkg/security/utils` package, I would suggest to move these files into `pkg/security/utils/container`, so that we have only one "tooling/utils" folder/package.

Reply:
I tried, but using pkg/security/utils in ptracer makes the binary goes from 6.8 to 26mo. Happy to discuss options!